src/core/server/metrics/metrics_service.ts

```diff
@@ -52,8 +65,16 @@ export class MetricsService
       this.refreshMetrics();
     }, config.interval.asMilliseconds());
 
+    // const metricsObservable: Observable<OpsMetrics>
     const metricsObservable = this.metrics$.asObservable();
 
+    analytics.registerEventType<OpsMetrics>({
+      eventType: 'core-ops_metrics',
+      schema: opsMetricsSchema,
+    });
+    // should we not be reporting the metrics during the start lifecycle?
+    this.reportOpsMetrics(analytics);
```

Comment:
nit: We can move this to a separate file with the method `setupOpsMetricsEventType`:

```ts
export function setupOpsMetricsEventType(analytics: AnalyticsServiceSetup, metricsObservable: Observable<OpsMetrics>) {
  analytics.registerEventType<OpsMetrics>({
    eventType: 'core-ops_metrics',
    schema: opsMetricsSchema,
  });
  metricsObservable.subscribe((metrics) => analytics.reportEvent('core-ops_metrics', { ...metrics });
}
```

I will make it easier to test and reduce the amount of code in the `metric_service` (especially because of the schema).